packages/datadog_flutter_plugin/e2e_test_app/ios/Podfile

```diff
@@ -33,8 +33,10 @@ target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
 
   # Datadog Pod Overrides
-  pod 'DatadogSDK', :git => 'https://github.com/DataDog/dd-sdk-ios', :tag => '1.23.0'
-  pod 'DatadogSDKCrashReporting', :git => 'https://github.com/DataDog/dd-sdk-ios', :tag => '1.23.0'
+  pod 'DatadogCore', :git => 'https://github.com/DataDog/dd-sdk-ios', :branch => 'develop'
```

Comment:
We probably want to point to specific version `2.2.1`

Reply:
During development I track develop. I have a script that removes these overrides when we go to release.